README.md

```diff
@@ -20,6 +20,10 @@ To Access my project locally please follow the following instructions;
 4. Once the download is complete you will be able to access my project locally.
 Here is my repository link: 
 
+## my live Portfolio link
+
+https://lyangend.github.io/MyPortfolio/
+
```

Comment:
- [ ] [_OPTIONAL_] Great job adding the live link to your readme, however, it's better to embed it in a text like [live link](URL)..... click on this [link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links) for further reference.